packages/react/src/components/Accordion/AccordionItem.js

```diff
@@ -134,15 +134,15 @@ AccordionItem.propTypes = {
   onHeadingClick: PropTypes.func,
 
   /**
-   * `true` to open the expando.
+   * `true` to open the expand.
    */
   open: PropTypes.bool,
 
   /**
-   * The callback function to render the expando button.
+   * The callback function to render the expand button.
    * Can be a React component class.
    */
-  renderExpando: PropTypes.func,
+  renderExpand: PropTypes.func,
```

Comment:
Would this be a breaking change?

Reply:
> Would this be a breaking change?

I was curious about that myself. This doesn't appear to be publicly exposed, so I thought no. Totally could be mis-reading that though.